pkg/nsx/services/mediator/mediator.go

```diff
@@ -67,8 +67,12 @@ func (serviceMediator *ServiceMediator) GetAvailableSubnet(subnetSet *v1alpha1.S
 			return *nsxSubnet.Path, nil
 		}
 	}
+	var tags []model.Tag
+	for k, v := range subnetSet.Labels {
+		tags = append(tags, model.Tag{Scope: common.String(k), Tag: common.String(v)})
+	}
 	log.Info("the existing subnets are not available, creating new subnet", "subnetList", subnetList, "subnetSet.Name", subnetSet.Name, "subnetSet.Namespace", subnetSet.Namespace)
```

Comment:
Here the func `CreateOrUpdateSubnet` is used to create **new** NSX subnet. Is it possible that the label on the subnetset changed so that we need to monitor the change and sync the tags? i.e. handle the NSX subnet's tag **update**?

Reply:
We mainly focus on the labels 	
   "default-vm-subnetset": common. LabelDefaultVMSubnet,
	"default-pod-subnetset": common. LabelDefaultPodSubnetSet, 
which won't change once created, so we can ignore the scenario when the labels updated?  @dantingl What's your opinion?